rules/windows/privilege_escalation_rogue_windir_environment_var.toml

```diff
@@ -81,16 +83,8 @@ query = '''
 registry where host.os.type == "windows" and event.type == "change" and
 registry.value : ("windir", "systemroot") and
 registry.path : (
-    "HKEY_USERS\\*\\Environment\\windir",
-    "HKEY_USERS\\*\\Environment\\systemroot",
-    "HKU\\*\\Environment\\windir",
-    "HKU\\*\\Environment\\systemroot",
-    "HKCU\\*\\Environment\\windir",
-    "HKCU\\*\\Environment\\systemroot",
-    "\\REGISTRY\\USER\\*\\Environment\\windir",
-    "\\REGISTRY\\USER\\*\\Environment\\systemroot",
-    "USER\\*\\Environment\\windir",
-    "USER\\*\\Environment\\systemroot"
+    "*\\Environment\\windir",
```

Comment:
Room to add volatile environment variables?

Reply:
Tagging @Samirbous as I'm not sure, but from what I saw, I don't think it could be used for the same purpose. I'll merge it as is and we can revisit it if needed